File: seoyeon/week14/2016년.py
def solution(a, b):
    li = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    last = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    month = a - 1
    count = sum(last[:month])-1
    return li[(count + b) % 7]

File: SiHyeon/week14/2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b)
{
    int month[] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};   // 1~12월 날 수
    string day[] = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"}; // 금요일부터 시작
    int index = 0;                                                    // 월의 날 수를 다 더한 값
    for (int i = 0; i < a - 1; i++)
        index += month[i];
    index = (index + b) % 7 - 1;
    return day[index == -1 ? 6 : index];
}